testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -2755,7 +2755,7 @@ public void listAllTypes() throws IOException {
     List<Map<String, String>> fields = (List<Map<String, String>>) response.get(0).get("fields");
     assertThat(fields.size()).isEqualTo(1);
     assertThat(fields.get(0).get("name")).isEqualTo("firstname");
-    assertThat(fields.get(0).get("typeDefinition")).isEqualTo("varchar");
+    assertThat(fields.get(0).get("typeDefinition")).isEqualTo("text");
```

Comment:
This is a weird one: I don't like "fixing" tests like this but as far as I can see, field was created as "text", and CQLsh/console returns "text" in such cases -- so I don't know how or why SGv1 would translate this into "varchar" somehow. I also don't know how I would reliably translate it either, given that it seems there is no 1-to-1 mapping between couple of textual types that all become "text".

So, open to suggestions here. I also note that our test coverage for UDT field/column types is rather... limited.

Reply:
Interesting. I can not help here, but I do notice that if this is wrong in the V1 we need a fix there asap as well?

Reply:
This: https://github.com/stargate/stargate/issues/1499

Reply:
That one is tricky so I will revert test change here to get this PR in (with 1 remaining "extra" fail, which was already failing for v2.0.0) and then tackle this test once we can resolve #1499.